tests/components/overkiz/test_config_flow.py

```diff
@@ -120,7 +120,7 @@ async def test_allow_multiple_unique_entries(hass: HomeAssistant) -> None:
     """Test config flow allows Config Flow unique entries."""
     MockConfigEntry(
         domain=DOMAIN,
-        unique_id="test2@testdomain.com",
+        unique_id=TEST_GATEWAY_ID2,
```

Comment:
Not 100% related to this PR, however while simplifying the test, I discovered that another test is still using the old unique id.

Is it ok to add this to this PR?

Reply:
Yes. Its the same topic